### PR TITLE
fix create-app to include setup for catalog-graph plugin

### PR DESCRIPTION
**Pull request by brethubbard**
## Hey, I just made a Pull Request!
There is an issue with the most recent release with the catalog-graph plugin because the template doesn't properly set up the plugin in App.tsx

This PR fixes that issue.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jhaals** [comment]
Thanks for addressing this @brethubbard, could you also make sure the changeset includes the changes required for existing apps similar to what we had for the previous change? See https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md#patch-changes

File: packages/create-app/templates/default-app/packages/app/src/App.tsx [hunk 1]
```diff
@@ -46,6 +50,9 @@ const app = createApp({
     bind(orgPlugin.externalRoutes, {
       catalogIndex: catalogPlugin.routes.catalogIndex,
     });
+    bind(catalogGraphPlugin.externalRoutes, {
+      catalogEntity: catalogPlugin.routes.catalogEntity,
```
Comment (Rugvip) [inline packages/create-app/templates/default-app/packages/app/src/App.tsx:54]: This is actually deprecated, so let's skip this. Change below is 👍  though

**Review by Rugvip — COMMENTED** [review]
Thank you!

**Comment by jhaals** [comment]
@brethubbard to get is in asap I applied a fix to your changes but I couldn't push to your branch(probably because it's master) so I created #9238

**Pull request closed without merging**